apps/pie-microsite/src/_11ty/shortcodes/tokensTable.js

```diff
@@ -130,6 +149,138 @@ const createTokensList = listElements => `<div class="c-tokensTable-row u-spacin
   ${listElements.join('')}
 </ul>`;
 
+/**
+ * Creates a tokens list for a given category
+ * @param {*} tokens 
+ * @param {string} path 
+ * @param {string} category 
+ * @param {string} tokenType 
+ * @returns 
```

Comment:
Would be good to detail the type of thing this returns and brief comment for each param